### PR TITLE
fix(cts): add waitForTask helpers tests [skip-bc]

### PR DESCRIPTION
**Pull request by millotp**
## 🧭 What and Why

Add all helpers to the spec and to the cts:
- [x] waitForTask
- [x] waitForAppTask

### CODE REVIEW

**Review by Fluf22 — COMMENTED** [review]
Looks good!
PHP doesn't seem to deserialize the mock server response correctly, though
And Java seems to consider your work as breaking

File: scripts/buildClients.ts [hunk 1]
```diff
@@ -15,6 +15,9 @@ async function buildLanguage(language: Language, gens: Generator[], buildType: B
     case 'csharp':
       await run('dotnet build --configuration Release', { cwd, language });
       break;
+    case 'go':
```
Comment (shortcuts) [inline scripts/buildClients.ts:18]: ooooplaaa

File: scripts/cli/index.ts [hunk 2]
```diff
@@ -147,9 +147,9 @@ ctsCommand
   .addArgument(args.language)
   .addArgument(args.clients)
   .option(flags.verbose.flag, flags.verbose.description)
-  .option('-e, --no-e2e', 'run the e2e tests, that requires internet connection')
-  .option('-c, --no-client', 'run the client tests')
-  .option('-r, --no-requests', 'run the requests tests')
+  .option('-e, --no-e2e', 'skip the e2e tests, that requires internet connection')
+  .option('-c, --no-client', 'skip the client tests')
+  .option('-r, --no-requests', 'skip the requests tests')
```
Comment (Fluf22) [inline scripts/cli/index.ts:152]: 😄😇 jk

```suggestion
  .option('-e, --skip-e2e', 'skip the e2e tests, that requires internet connection')
  .option('-c, --skip-client', 'skip the client tests')
  .option('-r, --skip-requests', 'skip the requests tests')
```

**Review by Fluf22 — APPROVED** [review]
✅

**Review by shortcuts — APPROVED** [review]
8th wonder of the world incoming

File: scripts/cli/index.ts (hunk 2 above)
Comment (millotp) [reply scripts/cli/index.ts:152]: sorry that doesn't work ahah

**Pull request merged**